deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-acs-admins-project
+rules:
+# ACS team can interact with ACS
+- apiGroups:
+  - platform.stackrox.io
+  - cloud.stackrox.io
+  resources:
+  - centrals
+  - securedclusters
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
```

Comment:
'patch' is not referenced in https://github.com/openshift/managed-cluster-config/pull/2085, only 'get'